mpu6050/perfomance_checking_program/main.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * main.h
+ *
+ *  Created on: 17 нояб. 2019 г.
```

Comment:
It is better to translate date to English and add author's email.

Reply:
I've fixed this remark